client/question/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -1384,4 +1384,61 @@ describe('FeedbackGeneratorService', function() {
       }
     );
   });
+
+  describe('prereqCheck errorLineNumber', function() {
+    it('should return null when there are no errors', function() {
+      var starterCode = [
+        'def myFunction(arg):',
+        '    return arg',
+        ''
+      ].join('\n');
+
+      var prereqFailure = PrereqCheckFailureObjectFactory.create(
+        PREREQ_CHECK_TYPE_WRONG_LANG, null, starterCode, 'notOp');
+      var feedback = FeedbackGeneratorService.getPrereqFailureFeedback(
+        prereqFailure);
+
+      expect(feedback.getErrorLineNumber()).toBeNull();
+    });
+
+    it('should return append the errorStr', function() {
```

Comment:
Could you please write this as a full sentence? I'm not sure what it's saying.

Reply:
Done.